website/sidebars.json

```diff
@@ -81,32 +81,66 @@
       "direct-manipulation"
     ],
     "The New Architecture": [
-      "new-architecture-intro",
+      "the-new-architecture/intro-2",
+      "the-new-architecture/why",
+      "the-new-architecture/use-app-template",
       {
         "type": "category",
-        "label": "Supporting in your Library",
+        "label": "Pillars",
         "collapsible": false,
         "collapsed": false,
         "items": [
-          "new-architecture-library-intro",
-          "new-architecture-library-android",
-          "new-architecture-library-ios"
+          "the-new-architecture/pillars",
+          "the-new-architecture/pillars-turbomodules",
+          "the-new-architecture/pillars-fabric-components"
         ]
       },
       {
         "type": "category",
-        "label": "Supporting in your App",
+        "label": "Backward Compatibility",
         "collapsible": false,
         "collapsed": false,
         "items": [
-          "new-architecture-app-intro",
-          "new-architecture-app-modules-android",
-          "new-architecture-app-modules-ios",
-          "new-architecture-app-renderer-android",
-          "new-architecture-app-renderer-ios"
+          "the-new-architecture/backward-compatibility",
+          "the-new-architecture/backward-compatibility-turbomodules",
+          "the-new-architecture/backward-compatibility-fabric-components",
+          "the-new-architecture/backward-compatibility-troubleshooting"
+        ]
+      },
+      {
+        "type": "category",
+        "label": "Migration",
+        "collapsible": false,
+        "collapsed": false,
+        "items": [
+          "new-architecture-intro",
```

Comment:
I see you retained this for the sake of not breaking links right? I feel that the `new-architecture-intro` could be moved to point to the new landign page.

Reply:
Yeah, in the effort to minimize changes and reuse what we have, I kept the old `The New Architecture` as the Migration guide. 
We could update this when we start working on the PRs that refactor the migration guide.